openapi/multi-yaml/index.yaml

```diff
@@ -59,6 +59,12 @@ tags:
       Payouts are processed each day at 11:30am US/Central time. A Platform can also configure each
       sub account to have an expedited payout priority. If this is enabled, the payout will be settled on the
       day the payout is generated. Otherwise, standard payouts will settle the next business day.
+  - name: Payout Holds
+    description: |
+      Payout Holds is a resource that allows platforms to temporarily hold or pause payouts for sub accounts.
+      This feature is used for risk management, compliance, or business rule enforcement.
+      Holds can be created automatically by the system (e.g., for first payments) or manually by platform administrators
```

Comment:
Same here, who are `platform administrators`? It sounds like it is some staff from the platform customer. 
If I understand correctly, those can only be created by the JustiFi team. 
Is that correct?